src/Cell.h

```diff
@@ -0,0 +1,108 @@
+#pragma once
+
+#include <cassert>
+#include <vector>
+
+#include "Config.h"
+#include "Gas.h"
+#include "Logging.h"
+#include "Parameters.h"
+
+
+namespace hydro_playground {
+  class Cell;
+
+  // template <int Dimensions>
+  class Grid {
```

Comment:
I'd rather have one file per class... could we move this to `Grid.h`?

Reply:
will do